src/MenuGraphr.ts

```diff
@@ -82,6 +87,7 @@ export class MenuGraphr implements IMenuGraphr {
         }
 
         this.GameStarter = settings.GameStarter;
+        this.ItemsHolder = new ItemsHoldr.ItemsHoldr();
```

Comment:
`this.GameStarter.ItemsHolder` already exists, just use that. You want your indices to be saved with the rest of the game data.